mix.exs

```diff
@@ -2,7 +2,7 @@ defmodule Ueberauth.Mixfile do
   use Mix.Project
 
   @source_url "https://github.com/ueberauth/ueberauth"
-  @version "0.10.4"
+  @version "0.10.5"
```

Comment:
These are dev and test dependencies. Do we need to change the version? As far as I can tell, we do not need to cut a release for it.

Reply:
I agree - that would be fine by me. I'll update the MR.